### DISCUSSION TITLE
Non-linear clearance

### DISCUSSION THREAD

**Discussion by OZaher** [Open-Systems-Pharmacology/Forum#1122] [Questions & Problems]
Hello all,
I am building a model for anticancer drug which has a non-linear clearance with time as it's clearance increase after multiple dosing.
This non-linearity is of physiological origin (related more to the metastasis happens in the patients) and is not related to enzyme induction.
Anyone know how can I add that change to the clearance of the drug after several doses.
Note: I thought of adding a new drug to be introduced at a time point after the initial dosing with the original drug but don't know how to make this or if that possible.
Thanks,
O.Zaher.

**Comment by SteTindal** [comment]
I have a similar problem.  How do I enter concentration dependant clearance for an IV dose?

**Reply by wilbertdew** [reply]
Concentration-dependant Clearance can be obtained in PK-Sim by using an enzyme with a saturable concentration as the metabolizing enzyme for your drug. This way you will have decreasing clearance with increasing drug concentrations. For the original question of @OZaher, most likely you need to manually change the clearance equations in MoBi.
